### ISSUE TITLE
updateProfile error.message

### ISSUE THREAD

**Issue by pepebe** [modxcms/Login#45]
I don't get anything in return for [[+error.message]].

The placeholder is listed in http://rtfm.modx.com/extras/revo/login/login.updateprofile and I expect it to return something like "an error has occured, please check your form entries".

I've checked core/components/login/controllers/web/UpdateProfile.php and found several things that look fishy.

Line 57:  'successMsgPlaceholder' => 'error.message', 
Line 260+: error.message only seems to be used for error set by preHooks. Validation doesn't seem to contribute to error.message.

I'd expect it to be returned by updateProfile whenever an error is returned.

I might be missing something here, so I decided to add my own placeholder just before line 202.

````
            $this->modx->toPlaceholder('update_error',1,$placeholderPrefix);
````

It will return a new placeholder [[+update_error]] which will return 1 if an error is presend.

P.S. I have also noticed that [[+login.update_success]] always comes with a prefix (login). Doesn't feel right as well.

**Comment by gadgetto** [comment]
Can confirm this!

**Comment by minagerges** [comment]
#27 duplicate, fix merged; c561d40
